scripts/test_action.sh

```diff
@@ -14,5 +14,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-yarn do src/www/build
+readonly TEST_DIR="${ROOT_DIR}/www"
```

Comment:
The `www` directory is used for packaging builds, maybe use `build/test` instead?

Reply:
Yeah, the `www` dir was being used for too many things.

I updated the test action and the jasmine config to use `build/test`.